app/controller.py

```diff
@@ -66,8 +67,10 @@ def changeToSaveAs(self):
     self.host.changeFocusTo(self.host.interactiveSaveAs)
 
   def doCommand(self, ch):
+    #Check the commandSet for the input with both its string and integer representation.
     self.savedCh = ch
-    cmd = self.commandSet.get(ch)
+    keyName = app.curses_util.cursesKeyName(ch)
+    cmd = self.commandSet.get(keyName) or self.commandSet.get(ch)
```

Comment:
I think using int and str together should work well. 

Though I just realized that the optimization I was hoping for will not pan out. I was thinking that a common ch value will be non-modified letters, like the letters I'm typing now in this sentence. Therefore we should check ints first, but I've coded it so that letters are entered as a fall through if no command is found. So, every normal letter typed is going to do a cursesKeyName() lookup, plus two .get() calls; before falling through to a normal letter input.

The calls to .get() are likely super cheap cpu-wise (based on prior tests), but a call into curses has been been slow in the past, so that's what I was trying to avoid. I may be 'optimizing too early'. Maybe I/we should try it and see if its a performance issue or not.